proposals/0229-simd.md

```diff
@@ -608,3 +599,46 @@ have been replaced with the "normal" arithmetic operations, matching `+`, `-`, `
 5. We have removed several operations whose naming was not totally satisfactory. We
 expect to add those operations and more in future additive proposals; only the types are
 necessary for ABI stability.
+
+6. Updates based on discussion on Swift-Evolution and with core team: I have eliminated
+`_`-prefixed protocol requirements, to make it somewhat more obvious how user types
+can be made to conform. This required some renaming to make the function of these
+associatedtypes more clear. I have also systematically removed `Vector` from the naming
+scheme, in order to remove confusion with C++-style vectors or "mathematical" vector
+structures that might be added at some future point. The resulting names are generally
+shorter, which is an nice benefit, and they now have a uniform `SIMD` prefix.
+
+  Protocols renamed:
+  
+  |  Old name | New name |
+  | --- | --- |
+  | `SIMDVector` | `SIMD` |
+  | `SIMDVectorizable` | `SIMDScalar` |
+  | `SIMDVectorStorage` | `SIMDStorage` |
+  | `SIMDMaskVector` | N/A* |
+
+  Associated types renamed:
+  
+  | Old name | New name |
+  | --- | --- |
+  | `Element` | `Scalar` |
+  | `_MaskElement` | `SIMDMaskScalar` |
+  | `_Vector${n}` | `SIMD${n}Storage` |
+  | `Mask` | N/A* |
+  | N/A* | `MaskStorage` |
+
+  Concrete types renamed:
+  
+  | Old name | New name |
+  | --- | --- |
+  | `Vector2<Scalar>` | `SIMD2<Scalar>` |
+  | `Vector3<Scalar>` | `SIMD3<Scalar>` |
+  | `Vector4<Scalar>` | `SIMD4<Scalar>` |
+  | `Vector8<Scalar>` | `SIMD8<Scalar>` |
+  | `Vector16<Scalar>` | `SIMD16<Scalar>` |
+  | `Vector32<Scalar>` | `SIMD32<Scalar>` |
+  | `Vector64<Scalar>` | `SIMD64<Scalar>` |
+  | N/A* | SIMDMask<Storage> |
```

Comment:
`SIMDMask<Storage>` needs backticks, otherwise the type parameter isn't rendered.